modules/mysql/read_replica.tf

```diff
@@ -16,7 +16,7 @@
 
 locals {
   replicas = {
-    for x in var.read_replicas : x.name => x
+    for x in var.read_replicas : "${local.master_instance_name}-replica${var.read_replica_name_suffix}${x.name}" => x
```

Comment:
This was done in order for the migration script to easily work on existing resource attributes. Anyone who's been using `master` directly recently will need to do a custom state migration manually.